client/handle_http.go

```diff
@@ -2160,7 +2185,7 @@ func (te *TransferEngine) walkDirDownloadHelper(job *clientTransferJob, transfer
 	}
 	infos, err := client.ReadDir(remotePath)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "Failed to PROPFIND remote directory")
```

Comment:
Same with this one, let's find a way to make this both something a user can interpret, and a developer can debug with.